Lib/test/test_builtin.py

```diff
@@ -304,7 +304,8 @@ def test_chr(self):
         self.assertEqual(chr(0x0010FFFF), "\U0010FFFF")
         self.assertRaises(ValueError, chr, -1)
         self.assertRaises(ValueError, chr, 0x00110000)
-        self.assertRaises((OverflowError, ValueError), chr, 2**32)
+        self.assertRaises(ValueError, chr, -2**100)
```

Comment:
Use `2**1000`. It is possible that once Python will run platform with 128-bit C long.